### PR TITLE
docs: add 2 manual steps for creating new versions

### PR DESCRIPTION
**Pull request by smoya**
**Description**

This PR adds 2 required manual steps for creating new versions.

### CODE REVIEW

**Comment by smoya** [comment]
Cc @jonaslagoni in case u want to 👀 this.

**Review by fmvilas — APPROVED** [review]

**Comment by derberg** [comment]
@smoya in case you are waiting for all codeowners, it is just a docs change, feel free to merge once you have one approval. If some codeowner doesn't like that change after it is merged, it is easy to revert, easier than some big change in a library

**Pull request merged**
